yvalenta/lib/server/server.rb

```diff
@@ -0,0 +1,46 @@
+require 'socket'
+require_relative 'connection'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.